src/components/card/index.tsx

```diff
@@ -1,25 +1,31 @@
 import React from 'react';
 
 type Props = {
-  className?: string
-  title?: React.ReactNode
-  titleClass?: string
-  children: React.ReactNode
-  actions?: React.ReactNode
+	className?: string
+	title?: React.ReactNode
+	headerClass?: string
+	children: React.ReactNode
+	actions?: React.ReactNode
 }
 
 export function Card(props: Props){
   return (
-    <div className={`card ${props.className}`}>
-      {props.title && (
-        <div className={`${props.actions ? 'd-flex align-items-center justify-content-between' : ''} ${props.titleClass}`}>
-          <h3 className="card-title">{props.title}</h3>
-          {props.actions}
-        </div>
-      )}
-      <div className="card-body">
-        {props.children}
-      </div>
-    </div>
-  )
+		<div className={`card ${props.className ?? ''}`}>
+			{
+				<div className={`card-header ${props.headerClass ?? ''}`}>
```

Comment:
this should handle
1. title only 
2. actions only 
3. both titles and actions
4. non of title and actions
please check with me when u finish this